woocommerce/woocommerce
# Update package lock files for API core tests to use Jest 25 instead of 27

### Changes proposed in this Pull Request:

This PR addresses an issue where the API core tests couldn't be ran with the following error:

```
TypeError: this.dom.getInternalVMContext is not a function
```

This was being caused by conflicting Jest versions (originally the package had Jest 27 but the rest of the packages use 25). Unfortunately the `package-lock.json` files still had the 27 version, which would throw the error above if the tests were attempted to be ran.

### How to test the changes in this Pull Request:

1. Verify the CI passes
2. On `trunk`, make sure you have the latest and run `npm install`.
3. Attempt to run the API tests from the top-level `$ woocommerce/` directory with `npm run test:api`
4. You should see the following error:

```
TypeError: this.dom.getInternalVMContext is not a function
```

5. Check out this branch and run `npm install`.
6. Now, run the API tests from the top-level `$ woocommerce/` directory with `npm run test:api` (optionally, you can [perform the setup and create a local `.env` file](https://github.com/woocommerce/woocommerce/tree/trunk/tests/e2e/api-core-tests#environment-variables))
7. The tests now should attempt to run. If it worked correctly, you should see the tests run and pass or `getaddrinfo ENOTFOUND undefined` if a local `.env` file wasn't created.
